common/src/main/java/com/codeforcommunity/enums/PrivilegeLevel.java

```diff
@@ -1,8 +1,10 @@
 package com.codeforcommunity.enums;
 
 public enum PrivilegeLevel {
-  STANDARD(1),
-  ADMIN(2);
+  NONE(-1),
+  STANDARD(0),
+  ADMIN(1),
+  SUPER_ADMIN(2);
```

Comment:
```suggestion
  STANDARD(0),
  ADMIN(1),
  SUPER_ADMIN(2);
```
I did some investigation and it looks like jOOQ uses the DB value as an index into the enum. So currently, in the DB 0 is returned in the JWT's privilegeLevel as `STANDARD(1)`, and 1 is returned as `ADMIN(2)`, and the frontend uses those values. 

If we remove the `NONE(-1)` case here, the enum values will align with the DB, which starts at 0. We'd still have `-1` represent a user not being authenticated on the frontend, but that should be set in default application state. Then, we can set the enum on the frontend to match the one here, going from 0-2.